blueprint/core/blueprint_ReactApplicationRoot.cpp

```diff
@@ -150,7 +150,9 @@ namespace blueprint
         DBG("");
         DBG(err.what());
 
-        errorText = std::make_unique<juce::AttributedString>(err.stack);
+        errorText = std::make_unique<juce::AttributedString>(err.what());
```

Comment:
Actually, if you refer to the comment [above](https://github.com/nick-thompson/blueprint/pull/188/files#r545477266), before this change here you would get a completely empty red error box.